apps/scale.go

```diff
@@ -107,6 +109,11 @@ func Scale(app string, sync bool, types []string) error {
 	res, err := c.AppsScale(app, scaleParams)
 	if err != nil {
 		if !utils.IsPaymentRequiredAndFreeTrialExceededError(err) {
+			reqestFailedError, ok := errors.ErrgoRoot(err).(*http.RequestFailedError)
+			if !ok || reqestFailedError.Code == 422 {
```

Comment:
I'm not sure I understand this conditino. Adding a comment to explain it may be a good idea